homeassistant/components/minecraft_server/config_flow.py

```diff
@@ -32,6 +32,10 @@ async def async_step_user(
         if user_input:
             address = user_input[CONF_ADDRESS]
 
+            # Abort config flow if device is already configured.
+            if await self._async_is_device_already_configured(address):
+                return self.async_abort(reason="already_configured")
```

Comment:
I think you can use the abort if entries match function for this

Reply:
Will have a look into it. Thanks 👍

Reply:
Great suggestion, I got the same result with less code, thanks again 👍